build/rewrite.go

```diff
@@ -54,17 +57,73 @@ func allowedSort(name string) bool {
 	return false
 }
 
-// Rewrite applies the high-level Buildifier rewrites to f, modifying it in place.
+// Rewrites controls the rewrites to be applied.
+//
+// If non-nil, the rewrites with the specified names will be run. If
+// nil, a default set of rewrites will be used that is determined by
+// the type (BUILD vs default starlark) of the file being rewritten.
+type Rewriter struct {
+	DisableRewrites                 *[]string
+	LabelArg                        *map[string]bool
+	Denylist                        *map[string]bool
+	SortableListArg                 *map[string]bool
+	SortDenyList                    *map[string]bool
+	SortAllowList                   *map[string]bool
+	NamePriority                    *map[string]int
+	StripLabelLeadingSlashes        *bool
+	ShortenAbsoluteLabelsToRelative *bool
+}
+
 func Rewrite(f *File) {
+	var rewriter = &Rewriter{
+		LabelArg:                        &tables.IsLabelArg,
+		Denylist:                        &tables.LabelDenylist,
```

Comment:
Why the renames? I'd expect the fields in the Rewriter struct to correspond 1:1 to the variables in the tables package.